static/src/javascripts/projects/common/modules/experiments/tests/utils/comment-blocker.js

```diff
@@ -0,0 +1,21 @@
+define([
+    'common/utils/config',
+    'lodash/collections/reduce'
+], function(
+    config,
+    reduce
+) {
+
+    var CommentBlocker = {};
+
+    CommentBlocker.hideComments = function(shortUrlSlug) {
+
+        var sUrlInt = shortUrlSlug !== null ? reduce(shortUrlSlug.split(''), function(sum, ch) {
```

Comment:
This makes me crosseyed haha. Interesting check! Maybe we could hook in a custom event somewhere and see what the actual difference between shown and not shown was in the end - not important for this test though!